<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoiseSimulationDialog</class>
 <widget class="QDialog" name="NoiseSimulationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout_6">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string> δ* source:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="deltaSourceXFoil">
            <property name="text">
             <string>XFoil calculation</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="deltaSourceBPM">
            <property name="text">
             <string>Original BPM δ* correlations</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="deltaSourceImport">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Import data</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widgetGridBPM" native="true">
          <layout class="QGridLayout" name="gridBPM">
           <item row="4" column="1">
            <widget class="QLineEdit" name="textOriginalAirfoilChordLength"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Original airfoil Chord length, C:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Original flow Mach Number, M:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>m</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Original flow velocity, U:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Type of transition:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Chord based Reynolds number, Rc:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>AOA, α:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="textAoa"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="textChordBased"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="textOriginalFlow"/>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="textOriginalFlowVelocity"/>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>[m/s]</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="comboTypeTransition">
             <item>
              <property name="text">
               <string>Fully turbulent</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Transaction flow</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetGridXFoilCalc" native="true">
          <layout class="QGridLayout" name="widgetGridXFoil">
           <item row="1" column="0" colspan="3">
            <widget class="QGroupBox" name="groupBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="title">
              <string>Interpolation scheme:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QCheckBox" name="checkLinear">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="baseSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Linear</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkNewtons">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>Newton´s</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkLagranges">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>Lagrange´s</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkSpline">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>Spline</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="4" column="0" colspan="3">
            <widget class="QLabel" name="label_29">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="textEddyConvectionMach"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>Eddy Convection Mach number (Mc):</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="textDScalingFactor"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>D* scaling factor:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="textDChordStation"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>D* at chord station:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string/>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>θe:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="textDirectivityGreek"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>φe:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="textDirectivityPhi"/>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../qblade.qrc">:/images/noise_3d_plate.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Select Directivity Factors (a picture with the reference system appears whenthe user clicks on the defaults for change – see picture below)</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>Simulation name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textSimulationName"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="verticalGroupBox_2">
       <property name="title">
        <string> OpPoint(s) for TE noise calculation:</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QListWidget" name="listOpPoints"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string/>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Select the directivity functions for high and low frequencies:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string>Low frequency</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBox_5">
          <property name="text">
           <string>High frequency</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string/>
       </property>
       <layout class="QFormLayout" name="formLayout_5">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Length of wetted Trailing-Edge (L):</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="textLengthWetted"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Distance from observer to TE (re):</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="textDistanceObserver"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Select acoustic weighting:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>C-Weighting</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>A-Weighting</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="radioButton_3">
          <property name="text">
           <string>B-Weighting</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioButton_4">
          <property name="text">
           <string>None</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string/>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_10">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Select TE noise source contributions desired and/or – more than one option may be selected here):</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Suction side of airfoil (attached flow)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string>Pressure side of airfoil (attached flow)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="text">
           <string>Separated flow on the suction side (high Reynolds flow)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textSimulationName</tabstop>
  <tabstop>listOpPoints</tabstop>
  <tabstop>textLengthWetted</tabstop>
  <tabstop>textDistanceObserver</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>deltaSourceXFoil</tabstop>
  <tabstop>deltaSourceBPM</tabstop>
  <tabstop>deltaSourceImport</tabstop>
  <tabstop>textAoa</tabstop>
  <tabstop>textChordBased</tabstop>
  <tabstop>textOriginalFlow</tabstop>
  <tabstop>comboTypeTransition</tabstop>
  <tabstop>textOriginalAirfoilChordLength</tabstop>
  <tabstop>textOriginalFlowVelocity</tabstop>
  <tabstop>textDChordStation</tabstop>
  <tabstop>checkLinear</tabstop>
  <tabstop>checkNewtons</tabstop>
  <tabstop>checkLagranges</tabstop>
  <tabstop>checkSpline</tabstop>
  <tabstop>textDScalingFactor</tabstop>
  <tabstop>textEddyConvectionMach</tabstop>
  <tabstop>textDirectivityGreek</tabstop>
  <tabstop>textDirectivityPhi</tabstop>
 </tabstops>
 <resources>
  <include location="../../qblade.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NoiseSimulationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>945</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NoiseSimulationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>951</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showEvent(QShowEvent)</slot>
 </slots>
</ui>
